workflow/common/common.go

```diff
@@ -93,6 +93,10 @@ const (
 	LabelKeyOnExit = workflow.WorkflowFullName + "/on-exit"
 	// LabelKeyArtifactGCPodHash is a label applied to WorkflowTaskSets used by the Artifact Garbage Collection Pod
 	LabelKeyArtifactGCPodHash = workflow.WorkflowFullName + "/artifact-gc-pod"
+	// LabelKeyCluster is a label applied to multi-cluster Workflows to identify which cluster they should be scheduled on
+	LabelKeyCluster = "/cluster"
+
+	LabelKeyHubWorkflowUID = "/workflow-uid"
```

Comment:
not sure if we want to prefix LabelKeyHubWorkflowUID with workflow name like the other labels

I figured LabelKeyCluster is something the user will use when creating the Workflow so didn't put it there (since they wouldn't have that information if `generateName` were used)